generators/setup-docker/templates/fabric-docker/commands-generated.sh

```diff
@@ -16,23 +16,36 @@ function installChaincodes() {
-      <% chaincode.channel.orgs.forEach(function (org) {
-           org.peers.forEach(function (peer) {
-      %>
-      printHeadline "Installing '<%= chaincode.name %>' on <%= chaincode.channel.name %>/<%= org.name %>/<%= peer.name %>" "U1F60E"
-      <% if(!networkSettings.tls) { -%>
-      chaincodeInstall "cli.<%= org.domain %>" "<%= peer.fullAddress %>" "<%= chaincode.channel.name %>" "<%= chaincode.name %>" "<%= chaincode.version %>" "<%= chaincode.lang %>" "$CHAINCODES_BASE_DIR/<%= chaincode.directory %>" "<%= rootOrg.ordererHead.fullAddress %>"
-      <% } else { -%>
-      chaincodeInstallTls "cli.<%= org.domain %>" "<%= peer.fullAddress %>" "<%= chaincode.channel.name %>" "<%= chaincode.name %>" "<%= chaincode.version %>" "<%= chaincode.lang %>" "$CHAINCODES_BASE_DIR/<%= chaincode.directory %>" "<%= rootOrg.ordererHead.fullAddress %>" "crypto/orderer-tlscacerts/tlsca.<%= rootOrg.domain %>-cert.pem"
-      <% } -%>
-      <% })}) -%>
+      <% chaincode.channel.orgs.forEach(function (org) { -%>
+        <% org.peers.forEach(function (peer) { -%>
+          printHeadline "Installing '<%= chaincode.name %>' on <%= chaincode.channel.name %>/<%= org.name %>/<%= peer.name %>" "U1F60E"
+          chaincodeInstall \
+            "cli.<%= org.domain %>" \
+            "<%= peer.fullAddress %>" \
+            "<%= chaincode.channel.name %>" \
+            "<%= chaincode.name %>" \
+            "<%= chaincode.version %>" \
+            "<%= chaincode.lang %>" \
+            "$CHAINCODES_BASE_DIR/<%= chaincode.directory %>" \
+            "<%= rootOrg.ordererHead.fullAddress %>" \
+            "<%= !networkSettings.tls ? '' : 'crypto/orderer-tlscacerts/tlsca.' + rootOrg.domain + '-cert.pem' %>"
+        <% }) -%>
+      <% }) -%>
 
       printItalics "Instantiating chaincode '<%= chaincode.name %>' on channel '<%= chaincode.channel.name %>' as '<%= chaincode.instantiatingOrg.name %>'" "U1F618"
-      <% if(!networkSettings.tls) { -%>
-      chaincodeInstantiate "cli.<%= chaincode.instantiatingOrg.domain %>" "<%= chaincode.instantiatingOrg.headPeer.fullAddress %>" "<%= chaincode.channel.name %>" "<%= chaincode.name %>" "<%= chaincode.version %>" "<%= chaincode.lang %>" "$CHAINCODES_BASE_DIR/<%= chaincode.directory %>" "<%= rootOrg.ordererHead.fullAddress %>" '<%- chaincode.init %>' "<%- chaincode.endorsement %>" "<%= chaincode.privateDataEnabled %>"
-      <% } else { -%>
-      chaincodeInstantiateTls "cli.<%= chaincode.instantiatingOrg.domain %>" "<%= chaincode.instantiatingOrg.headPeer.fullAddress %>" "<%= chaincode.channel.name %>" "<%= chaincode.name %>" "<%= chaincode.version %>" "<%= chaincode.lang %>" "$CHAINCODES_BASE_DIR/<%= chaincode.directory %>" "<%= rootOrg.ordererHead.fullAddress %>" '<%- chaincode.init %>' "<%- chaincode.endorsement %>" "<%= chaincode.privateDataEnabled %>" "crypto/orderer-tlscacerts/tlsca.<%= rootOrg.domain %>-cert.pem"
-      <% } -%>
+      chaincodeInstantiate \
```

Comment:
hm... not sure about this one. `commands-generated.sh` tends to be big file. We also can install instantiate chaincodes on multiple channels. I would keep it as one liner as it's easier to reason about whole network setup that way.